src/tests/functional/plugin/shared/include/low_precision_transformations/batch_to_space_transformation.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <memory>
+
+#include "shared_test_classes/base/low_precision_transformations/layer_transformation.hpp"
+#include "lpt_ngraph_functions/common/fake_quantize_on_data.hpp"
+
+
+namespace LayerTestsDefinitions {
+class BatchToSpaceTransformationParam {
+public:
+    ngraph::PartialShape input_shape;
+    std::vector<size_t> block_shape;
+    std::vector<size_t> crops_begin;
+    std::vector<size_t> crops_end;
+    ngraph::builder::subgraph::FakeQuantizeOnData fake_quantize;
+    std::string layer_type;
+    std::string expected_kernel_type;
+};
+
+typedef std::tuple<
+    ngraph::element::Type,
+    std::string,
+    ngraph::pass::low_precision::LayerTransformation::Params,
```

Comment:
If I am not mistaken, we don't use `LayerTransformation::Params` in this test. Can we not include it to `BatchToSpaceTransformationParam` list maybe? The same for s2b tests

Reply:
done, thanks